content/zh-cn/docs/reference/glossary/cluster.md

```diff
@@ -25,17 +23,17 @@ aka:
 tags:
 - fundamental
 - operation
---- 
 -->
 
 <!-- 
 A set of worker machines, called {{< glossary_tooltip text="nodes" term_id="node" >}},
 that run containerized applications. Every cluster has at least one worker node.
 -->
-集群是由一组被称作{{< glossary_tooltip text="节点（node）" term_id="node" >}}的机器组成，
-这些节点上会运行由 Kubernetes 所管理的容器化应用。
-且每个集群至少有一个工作节点。
+一组工作机器，称为 {{< glossary_tooltip text="节点" term_id="node" >}}，
```

Comment:
```suggestion
一组工作机器，称为{{< glossary_tooltip text="节点" term_id="node" >}}，
```

Reply:
This is a reasonable suggestion, why not fixed?